setup.cfg

```diff
@@ -2,3 +2,37 @@
 ignore =
     *.cfg
     .travis.yml
+
+[flake8]
+exclude =
+    bootstrap-buildout.py,
+
+ignore =
+    N801,
+    N802,
+    N803,
+    N805,
+    N806,
+    N812,
+    T000,
```

Comment:
As part of the other comment: flake8-naming is not part of the default list of flake8 add-ons, so I would remove these.

Eventually we might want to enable it, but for now it will be too much noise.